schema/README.md

```diff
@@ -0,0 +1,95 @@
+# Simple Schema for Observability
+
+## Background
+Observability is the ability to measure a system’s current state based on the data it generates, such as logs, metrics, and traces. Observability relies on telemetry derived from instrumentation that comes from the endpoints and services.
+
+Observability telemetry signals (logs, metrics, traces, alerts) arriving from the system would contain all the necessary information needed to observe and monitor.
+
+Modern application can have a complicated distributed architecture that combines cloud native and microservices layers. Each layer produces telemetry signals that may have different structure and information.
+
+Using Observability telemetry schema we can organize, correlate and investigate system behavior in a standard and well-defined manner.
+
+Observability telemetry schema defines the following components - logs, traces, metrics and alerts.
+
+Logs provide comprehensive system details, such as a fault and the specific time when the fault occurred. By analyzing the logs, one can troubleshoot code and identify where and why the error occurred.
+
+Traces represent the entire journey of a request or action as it moves through all the layers of a distributed system. Traces allow you to profile and observe systems, especially containerized applications, serverless architectures, or microservices architecture.
+
+Metrics provide a numerical representation of data that can be used to determine a service or component’s overall behaviour over time.
+
+In many occasions, correlate between the logs, traces and metrics is mandatory to be able to monitor and understand how the system is behaving. In addition, the distributed nature of the application produces multiple formats of telemetry signals arriving from different components ( network router, web server, database)
+
+For such correlation to be possible the industry has formulated several protocols ([OTEL](https://github.com/open-telemetry), [ECS](https://github.com/elastic/ecs), [OpenMetrics](https://github.com/OpenObservability/OpenMetrics), [Alerts]()) for communicating these signals - the Observability schemas.
```

Comment:
Alerts ref is not valid - do we even support it today? If not can we remove it and say "currently" and add it back later?

Reply:
good point